src/source/examples/put-campaign-linkedads-v1-example.rst

```diff
@@ -0,0 +1,20 @@
+.. code-block:: javascript
+
+    PUT /api/sellside/campaign/123/linkedAds
+    Accept: application/sellside.campaign.linkedad-v1+json; charset=utf-8
+    Content-Type: application/sellside.campaign.linkedad-v1+json; charset=utf-8
+
+    {
+       "adId": 12315452,
+       "cpc": 3,
```

Comment:
earlier you said that if you supply `autobid:true` that you should not supply `cpc`; however here we see that that's possible, it just gets ignored. Make sure we either tell that the field is ignored, or that we do validation and throw BAD REQUEST.

Reply:
it's bad request, good point, will change (this is also unexposed yet)